src/participate-sora-governance.md

```diff
@@ -0,0 +1,73 @@
+
+![](/.gitbook/assets/participate-sora-governance-overview.png)
+
+2. With XOR in your account, select the democracy referenda you are interested in. You can click the downward arrow to expand the motion details. In some cases, you will find an address linking it to the [SORA Accounting project on Github](https://github.com/orgs/sora-xor/projects/24/views/1) that has an overview and description of the referenda, if it pertains to credit creation on SORA.
+
+![](/.gitbook/assets/participate-sora-governance-expand-referenda.png)
+
+3. To vote on the referenda, click on the vote button. A pop-up will appear with your account information. If you click on the account name, you can cycle through your accounts on the wallet. Pick the one you will use to vote.
+
+![](/.gitbook/assets/participate-sora-governance-account-selection.png)
+
+4. After selecting the account, you will see the voting balance which you can use to allocate the weight of your vote. Ensure you have enough balance left unlocked to pay the transaction fees. The vote value field will be automatically populated if you already have XOR balance locked.
+
+![](/.gitbook/assets/participate-sora-governance-vote-value.png)
+
+::: info
+You can use funds that have been locked by nominating a validator or are locked in another governance proposal to vote. Once the funds are locked, you must wait a set amount of time before unlocking them in the Polkadot.js interface.
+:::
+
+5. Choose a conviction for your vote; this can go from a 1x voting balance locked for 30 days to a 6x voting balance for 960 days. Finally, vote Aye or Nay on the proposal.
+
+![](/.gitbook/assets/participate-sora-governance-vote-conviction.png)
+
+::: info
+If the proposal passes and you vote yes, your funds will lock for the allocated amount of time based on its conviction; likewise, if the proposal does not pass and you vote no, your funds will also remain locked. Vice versa, if you voted no and it passes, your funds will not be locked, idem if you voted aye and the proposal does not pass.
+:::
+
+![](/.gitbook/assets/participate-sora-governance-vote-confirmation.png)
+
+6. Your wallet/signer will pop up requesting you to confirm your vote, approve this transaction and once it is confirmed, **congratulations, you have just voted on SORA!**
+
+### How to vote in the SORA Kusama and SORA Polkadot Parachains
+
+In practice, voting for governance referenda on SORA’s parachains is the same. The only difference is that you need to bridge XOR over to the parachains using the HASHI bridge set to Polkadot or Kusama network and switch networks within the Polkadot.js interface.
+
+From the top right corner of the interface, click on the downward arrow > Click on Polkadot/Kusama & Parachains > Scroll down to SORA > Click on Switch.
```

Comment:
I think using `>` here to denote the steps is inconsistent with how it's done in the rest of the docs.

Reply:
Are there any suggested symbols to use instead?